datalad/cmdline/main.py

```diff
@@ -557,6 +560,12 @@ def main(args=None):
                 # had no code defined
                 sys.exit(exc.code if exc.code is not None else 1)
             except Exception as exc:
+                title = "DataLad Cmd Error: %s" % exc.__class__.__name__
+                submit_helpme(
```

Comment:
Myself: need to check if actually current UI  is interactive (UI while running tests is not, so that would avoid setting envvar in Cis). We might actually make env var such that it could force helpme in noon interactive sessions to assist filling issues in such cases

Reply:
And ideally in the longer run would be great if helpme could use our ui interface to ask questions etc

Reply:
Definitely! We can handle this in another scoped PR / issue.